app/views/staff/transactions/edit.html.haml

```diff
@@ -0,0 +1,11 @@
+=content_for :page_title_prefix, t("page_title.transaction.edit")
+
+= link_to t("generic.link.back"), fund_path(@fund), class: "govuk-back-link"
+%main.govuk-main-wrapper#main-content{ role: "main" }
+  .govuk-grid-row
+    .govuk-grid-column-two-thirds
+      %h1.govuk-heading-xl
+        = t("page_title.transaction.edit")
+
+      = form_with model: @transaction, url: fund_transaction_path(@fund), method: :patch do |f|
```

Comment:
I've learnt that we don't need the `url` or `method` fields for the new form builder so you should be able to remove these without impact 👍

Reply:
If I remove them, I get an error ` undefined method `transactions_path'` - the path is `fund_transactions_path`, so I think it needs to stay in.